sdk/iot/Azure.Iot.Hub.Service/src/BulkIfMatchPrecondition.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.Iot.Hub.Service
+{
+    /// <summary>
+    /// The condition on which each operation in a bulk operation will execute.
+    /// </summary>
+    public enum BulkIfMatchPrecondition
+    {
+        /// <summary>
+        /// Perform this operation regardless of if the provided resource matches the service's representation
```

Comment:
Q - the `matches` in the description refers to the Etag match, correct? Why not say:
` Perform this operation regardless of if  the provided entity's ETag matches the service's ETag.`?

Reply:
Sure, but with ```resource``` instead of ```entity``` for consistency